exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Function.kt

```diff
@@ -182,7 +182,7 @@ class Min<T : Comparable<T>, in S : T?>(
 /**
  * Represents an SQL function that returns the maximum value of [expr] across all non-null input values, or `null` if there are no non-null values.
  */
-class Max<T : Comparable<T>, in S : T?>(
+class Max<T : Any, in S : T?>(
```

Comment:
These changes were only made because our tests had use cases like `TestTable.id.max()`.

@e5l @obabichevjb This now leads to the question, what about other restricted functions, like `Avg()`, which only accepts Comparable column types?
Should it still be possible to call `avg()` on an entityID column like `TestTable.id.avg()` or does that use case make no sense?

Reply:
That’s a good question. At first glance, it seems like it shouldn't be `Comparable`, especially for `Max` functions, if it’s only comparable on the database side. I can’t think of a specific example right now, but imagine something that can have `Max` applied to it in the database but doesn’t have a comparable representation in Kotlin. 

Since we get from SQL just single value and don't compare it on client side, it could be non-comparable.